### PR TITLE
modified:   chapter_installation/index.md

### PR DESCRIPTION
**Pull request by xinetzone**
Modify the expression problem

### CODE REVIEW

**Comment by mli** [comment]
Job d2l-zh/PR-584/1 is complete. 
Check the results at http://preview.d2l.ai/d2l-zh/PR-584/

**Comment by xinetzone** [comment]
Can it be merged?

**Comment by mli** [comment]
Job d2l-zh/PR-584/2 is complete. 
Check the results at http://preview.d2l.ai/d2l-zh/PR-584/

File: chapter_installation/index.md [hunk 1]
```diff
@@ -1,9 +1,9 @@
 # 安装
 :label:`chap_installation`
 
-为了让您开始运行并获得动手学习体验，我们需要为您设置一个运行 Python、Jupyter 笔记本电脑、相关库以及运行本书所需的代码的环境。
+为了让您起步并获得动手学习的体验，我们需要为您设置一个运行 Python、Jupyter Notebook、相关库以及运行本书所需的代码的环境。
```
Comment (xiaotinghe) [inline chapter_installation/index.md:4]: “您”->“你”，不需要使用尊称。
“设置”->“配置”

File: chapter_installation/index.md [hunk 2]
```diff
@@ -1,9 +1,9 @@
...
 
-## 安装米尼康达
+## 安装 Miniconda
 
 最简单的方法就是安装 [Miniconda](https://conda.io/en/latest/miniconda.html)。Python 3.x 版本是必需的。如果已安装 conda，则可以跳过以下步骤。从网站下载相应的 Miniconda sh 文件，然后使用 `sh <FILENAME> -b` 从命令行执行安装。对于 macOS 用户：
```
Comment (xiaotinghe) [inline chapter_installation/index.md:8]: 这里依赖Python 3.x 版本

File: chapter_installation/index.md [hunk 3]
```diff
@@ -31,14 +31,14 @@ sh Miniconda3-latest-Linux-x86_64.sh -b
 conda create --name d2l -y
 ```
 
-## 下载 D2L 笔记本电脑
+## 下载 D2L Notebook
 
-接下来，我们需要下载这本书的代码。您可以点击任何 HTML 页面顶部的 “所有笔记本” 选项卡下载并解压代码。或者，如果您有 `unzip`（否则运行 `sudo apt install unzip`）可用：
+接下来，我们需要下载这本书的代码。您可以点击任何 HTML 页面顶部的 “Jupyter 笔记本文件” 选项卡下载并解压代码。或者，如果您有 `unzip`（否则运行 `sudo apt install unzip`）可用：
```
Comment (xiaotinghe) [inline chapter_installation/index.md:36]: “您”->“你”
“如果您有 `unzip`（否则运行 `sudo apt install unzip`）可用：”->“如果已经安装`unzip`（通过运行 `sudo apt install unzip`安装）可以执行下面的命令”

File: chapter_installation/index.md [hunk 4]
```diff
@@ -58,21 +58,25 @@ conda install python=3.7 pip -y
...
 :begin_tab:`pytorch`
 ```bash
-pip install torch==1.5.1 torchvision -f https://download.pytorch.org/whl/torch_stable.html
+pip install torch torchvision -f https://download.pytorch.org/whl/torch_stable.html
 ```
 :end_tab:
 
 :begin_tab:`tensorflow`
-您可以通过以下方式安装具有 CPU 和 GPU 支持的腾讯流：
+您可以通过以下方式安装，提供 CPU 和 GPU 支持：
```
Comment (xiaotinghe) [inline chapter_installation/index.md:76]: 你可以通过以下命令安装具有 CPU 和 GPU 支持的tensorflow：

**Comment by mli** [comment]
Job d2l-zh/PR-584/3 is complete. 
Check the results at http://preview.d2l.ai/d2l-zh/PR-584/

**Comment by mli** [comment]
Job d2l-zh/PR-584/4 is complete. 
Check the results at http://preview.d2l.ai/d2l-zh/PR-584/

**Comment by xinetzone** [comment]
Complete all modifications.

**Comment by xinetzone** [comment]
Can it be merged?

**Comment by mli** [comment]
Job d2l-zh/PR-584/5 is complete. 
Check the results at http://preview.d2l.ai/d2l-zh/PR-584/

**Comment by xiaotinghe** [comment]
Thanks

**Pull request merged**